File: 프로그래머스/0/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.java
class Solution {
    public String solution(String my_string, String alp) {
        char [] ch = new char[my_string.length()];
        for(int i = 0; i < ch.length; i++){
            ch[i] = my_string.charAt(i);
            if(ch[i] == alp.charAt(0)){
                ch[i] -=32;
            }
        }
        String answer = String.valueOf(ch);
        return answer;
    }
}

File: 프로그래머스/0/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.cs
using System;

class Solution {
    public String solution(String my_string, String alp) {
        char [] ch = new char[my_string.Length];
        for(int i = 0; i < ch.Length; i++){
            ch[i] = my_string[i];
            if(ch[i] == alp[0]){
                ch[i] -=(char)32;
            }
        }
        String answer = new String(ch);
        return answer;
    }
}